src/pages/gen2/build-a-backend/data/connect-to-existing-database/index.mdx

```diff
@@ -0,0 +1,415 @@
+
+Next, import the schema into your data resource file to deploy to the cloud.
+
+```ts title="amplify/data/resource.ts"
+import { a, defineData } from '@aws-amplify/backend';
+// highlight-next-line
+import { schema as rdsSchema } from './schema.rds.ts';
+
+// Relational database sources can coexist with DynamoDB tables managed by Amplify.
+const schema = a.schema({
+  Todo: a.model({
+    content: a.string(),
+    isDone: a.boolean(),
+  }),
+});
+
+// Use the `.combine()` operator to stitch together the models backed by DynamoDB
+// and the models backed by Postgres or MySQL databases.
+// highlight-start
+const combinedSchema = a.schema.combine([schema, rdsSchema]);
+// highlight-end
+
+export type Schema = typeof schema;
+export const data = defineData({
+  // highlight-next-line
+  schema: combinedSchema,
+});
+```
+
+Once your cloud sandbox has updated, you should be able to call any CRUDL operations for your relational database.
+
+```bash
+npx amplify sandbox
+```
+
+### Apply authorization rules
+
+```ts title="amplify/data/resource.ts"
+import { a, defineData } from '@aws-amplify/backend'
+import { schema as rdsSchema } from './schema.rds.ts'
+
+// Add an authorization rule to the schema
+// highlight-next-line
+rdsSchema.models.Todo.authorization([a.allow.public()])
+
+const schema = a.schema({
+  Todo: a.model({
+    content: a.string()
+    isDone: a.boolean()
+  })
+})
+
+const combinedSchema = a.schema.combine([
+  schema,
+  rdsSchema
+])
+
+export type Schema = typeof schema
+export const data = defineData({
+  schema: combinedSchema
+})
+```
+
+### Rename inferred model and field names
+
+```ts title="amplify/data/resource.ts"
+import { a, defineData } from '@aws-amplify/backend'
+import { schema as rdsSchema } from './schema.rds.ts'
+
+// Rename models or fields to be more idiomatic for frontend code
+// highlight-start
+rdsSchema.models.todos.renameTo("Todo")
+rdsSchema.models.todos.fields.title.renameTo("content")
+// highlight-end
+
+const schema = a.schema({
+  Todo: a.model({
+    content: a.string()
+    isDone: a.boolean()
+  })
+}).authorization([a.allow.public()])
+
+const combinedSchema = a.schema.combine([
+  schema,
+  rdsSchema
+])
+
+export type Schema = typeof schema
+export const data = defineData({
+  schema: combinedSchema
+})
+```
+
+### Invoke SQL statements with custom queries and mutations
+
+```ts title="amplify/data/resource.ts"
+import { a, defineData } from '@aws-amplify/backend'
+import { schema as rdsSchema } from './schema.rds.ts'
+
+// Add custom mutations or queries that execute SQL statements
+// highlight-start
+rdsSchema.addQueries({
+  searchTodoBasedOnGeolocation: a.query()
+    .arguments({
+      lat: a.float(),
+      long: a.float(),
+      radiusInMeters: a.float()
+    })
+    .returns(a.ref("Todo").array())
+    .handler(sql`
```

Comment:
I'm unsure what the `sql` template string function is doing for us here. Is this a function that our library provides that should be added to the import above? Is this coming from somewhere else? What behavior does it provide?

Is this most similar to one of the following?
- https://github.com/andywer/squid?tab=readme-ov-file
- https://www.npmjs.com/package/sql-template-strings
- https://www.npmjs.com/package/sql-template-tag

I thought I might find this idiom in Prisma, but they appear to support raw sql as plain strings.